client/src/global/components/Annotation/Annotation/TextContent/FromNodes/Wrapper.js

```diff
@@ -1,15 +1,21 @@
 import React from "react";
 import Collapse from "global/components/Collapse";
 import * as Styled from "./styles";
+import { useTranslation } from "react-i18next";
+import IconComposer from "global/components/utility/IconComposer";
 
 export default function FromNodesWrapper({ children, overlayLight }) {
+  const { t } = useTranslation();
   return (
     <Collapse stubHeight={200}>
+      <Styled.Content focusOnVisible>
+        <blockquote>{children}</blockquote>
+        <Styled.Overlay $light={overlayLight} />
+      </Styled.Content>
       <Styled.Toggle>
-        <Styled.Content>
-          {children}
-          <Styled.Overlay $light={overlayLight} />
-        </Styled.Content>
+        <span className="toggle-show-label">{t("actions.show")}</span>
+        <span className="toggle-hide-label">{t("actions.hide")}</span>
```

Comment:
Let's adjust these a bit. "Expand" and "collapse" seem like more accurate descriptions of what the toggle does. And for expanding, let's do `<span>Expand <span class="screen-reader-text">passage</span></span>`. That will provide a bit more info to SR users. The region is also labelled by the toggle, so it'll be doubly useful.